hydra/main.py

```diff
@@ -1,17 +1,46 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import copy
 import functools
+import pickle
+import warnings
+from pathlib import Path
 from textwrap import dedent
-from typing import Any, Callable, Optional
+from typing import Any, Callable, List, Optional
 
-from omegaconf import DictConfig
+from omegaconf import DictConfig, open_dict, read_write
 
 from ._internal.deprecation_warning import deprecation_warning
 from ._internal.utils import _run_hydra, get_args_parser
+from .core.hydra_config import HydraConfig
+from .core.utils import _flush_loggers, configure_log
 from .types import TaskFunction
 
 _UNSPECIFIED_: Any = object()
 
 
+def _get_rerun_conf(file_path: str, overrides: List[str]) -> DictConfig:
+    msg = "Experimental rerun CLI option."
+    warnings.warn(msg, UserWarning)
+    file = Path(file_path)
+    if not file.exists():
+        raise ValueError(f"{file} does not exist!")
+
+    if len(overrides) > 0:
+        msg = "Config overrides are not supported as of now."
+        warnings.warn(msg, UserWarning)
+
+    with open(str(file), "rb") as input:
+        config = pickle.load(input)  # nosec
+        configure_log(config.hydra.job_logging, config.hydra.verbose)
+        HydraConfig.instance().set_config(config)
+        task_cfg = copy.deepcopy(config)
+        with read_write(task_cfg):
+            with open_dict(task_cfg):
+                del task_cfg["hydra"]
+        assert isinstance(task_cfg, DictConfig)
+        return task_cfg
```

Comment:
I think there is no need to indent beyond the call to `pickle.load`.
```suggestion
    with open(str(file), "rb") as input:
        config = pickle.load(input)  # nosec
    configure_log(config.hydra.job_logging, config.hydra.verbose)
    HydraConfig.instance().set_config(config)
    task_cfg = copy.deepcopy(config)
    with read_write(task_cfg):
        with open_dict(task_cfg):
            del task_cfg["hydra"]
    assert isinstance(task_cfg, DictConfig)
    return task_cfg
```